src/Synth/ADnote.cpp

```diff
@@ -1796,6 +1824,9 @@ inline void ADnote::ComputeVoiceOscillatorFrequencyModulation(int nvoice,
  */
 inline void ADnote::ComputeVoiceOscillatorWaveTableModulation(int nvoice, FMTYPE FMmode)
 {
+  Voice& vce = NoteVoicePar[nvoice];
```

Comment:
Can you fix the indentation here, please?

Reply:
done